README.md

```diff
@@ -5,13 +5,61 @@ This repository is used to build the base container for gridappsd.
 It also includes a few scripts one to create the releases and another to create a blazegraph container.
 
 ./scripts/release/create_release.sh - Requires github username and token.  
-Release process:
-  * List open pull requests
-  * Create the release/$VERSION from develop
-  * Test the releases/$VERSION
-  * From the releases/$VERSION branch, create the pull request to master
-  * Assign and close the pull requests
-  * After the pull request's are approved then create the Tagged Releases
+
+To create a release, the create_release.sh script is run multiple times, between each run there are manual testing and validation steps.
+
+1.  Clone the build repository 
+```
+git clone https://github.com/GRIDAPPSD/gridappsd-docker-build
+```
+
+1.  Clone the gridappsd-docker repository
+```
+git clone https://github.com/GRIDAPPSD/gridapspd-docker
+```
+
+1.  Edit the create_release.sh script to update the version and add GitHub username and token.
+```
+cd gridappsd-docker-build/scripts/release
+vi create_release.sh
```

Comment:
I assume there is a VERSION named in here...can we have that in here as what we should change

Reply:
I will update to make that more clear.